### ISSUE TITLE
Potential accessibility issue with the tab component 

### ISSUE THREAD

**Issue by Ciandelle** [alphagov/govuk-design-system#2719]
## Description of the issue
 unable to tab-focus and use arrow keys to navigate the data held inside the tab, and as such for a screen reader, this content is inaccessible.

## Environment (where applicable)
Using NVDA screen reader and using Chrome to browse.

## Next steps
We are aware of this as a potential issue and are planning to look into it further

**Comment by davidc-gds** [comment]
Hi @Ciandelle, have you tried using the arrow keys to navigate between tabs?

The [tabs component](https://design-system.service.gov.uk/components/tabs/) is built as an 'ARIA widget', and ARIA widgets follow the general rule that:
- you tab once to enter the ARIA widget
- inside the widget, you use arrow keys to navigate

This is to make it possible for people navigating via keyboard to tab past a widget they don't want to interact with, without getting lost in the widget's options.

**Comment by davidc-gds** [comment]
Ohh @Ciandelle sorry, is this the bug we got through our support email? I think it was specifically that they couldn't use tab and arrow keys to properly navigate a table inside a tab component. My bad, that is something we'll need to investigate.

**Comment by Ciandelle** [comment]
@davidc-gds Hey, yes it was, would be be helpful for me to put the full details from Zendesk in this issue so that it makes a bit more sense?

**Comment by davidc-gds** [comment]
I tested this with NVDA on Chrome, and didn't find any issues with our implementations.

Here's the example frame I tested: https://design-system.service.gov.uk/components/tabs/default/index.html

### Test results
Here's the [full NVDA testing results in a spreadsheet](https://docs.google.com/spreadsheets/d/1pCrJ1z_pi5PpvoKg8kLAWeplJLWUxifN_Qd7L5jaQ5I/edit?usp=sharing).
- Successfully navigated to the start of the tab group with tab key
- Successfully navigated between tabs with left and right arrow keys
- Successfully exited tab-list navigation area with 'esc' key, and entered tab content with down arrow
- Successfully navigated to the table in the tab-panel using either the tab key or the down arrow
- Successfully navigated to the table from the top of the example using the 'T' NVDA keyboard shortcut. The shortcut did not list the hidden tables on unselected tabs, which is the correct and expected functionality.
- Successfully navigated inside the table inside the tab by using NVDA's table navigation keys (on Mac it's 'control' + 'option' + arrow keys)

For our implementation on the GOV.UK Design System, everything appeared to behave as expected in NVDA.

### Technical test details
**Assistive technology:** NVDA Version 2022.4 (latest)
**Browser:** Chrome
**Browser version:** 111 (latest)
**Operating System:** Windows 10 (Assistiv Labs)

**Comment by 36degrees** [comment]
Is this potentially the same as https://github.com/alphagov/govuk-frontend/issues/860?

Unless I'm missing something, I disagree with the conclusion that the content is inaccessible – though we could be doing better here.

**Comment by davidc-gds** [comment]
@Ciandelle it looks like this is now closed. Should we perhaps replay back to the original email we received, with the results of the tests I quickly ran?
https://github.com/alphagov/govuk-design-system/issues/2719#issuecomment-1499319860

**Comment by Ciandelle** [comment]
@davidc-gds Owen went back to the user already :) we can provide them with this if they come back to us again
